examples/mnist/notebook.jl

```diff
@@ -239,18 +246,21 @@ savefig(joinpath(DIR, "loss.png"))
 n_epochs =  length(losses)
 n_parameters = div(length(parameter_means), n_epochs)
 parameter_means2 = reshape(copy(parameter_means), n_parameters, n_epochs)'
-plot(epochs, parameter_means2,
-     title="Flux parameter mean weights",
-     xlab = "epoch")
+plot(
+    epochs,
+    parameter_means2,
+    title="Flux parameter mean weights",
+    xlab = "epoch",
+)
 
-# **Note.** The the higher the number, the deeper the chain parameter.
+# **Note.** The higher the number, the deeper the chain parameter.
```

Comment:
I don't get in what sense its deeper because perhaps this is using the same number of hidden layers regardless to epochs.

Reply:
We are reporting the mean value of weights for different layers. The higher the index, the deeper the layer we are averaging. 

Added a comment to clarify, but let me know if this is still not clear or if you believe my assertion is not correct.

Reply:
OK. I hadn't seen the plot or noticed the `div` operation when I commented. Now I understand that the higher the index in the means list, the deeper the layer. I think I initially understood "number" to be the epoch number.

Those we can either just assume users will conclude that "number" is the index in the plot or clarify it.

Reply:
@ablaom this comment.